docs/two-phase-commit-transactions.md

```diff
@@ -195,6 +195,111 @@ Similar to `prepare()`, you can call `validate()` in the coordinator/participant
 Currently, you need to call `validate()` when you use the `Consensus Commit` transaction manager with `EXTRA_READ` serializable strategy in `SERIALIZABLE` isolation level.
 In other cases, `validate()` does nothing.
 
+### Handle Exceptions
+
+Let's look at the following example code to see how to handle exceptions in Two-phase commit transactions.
+
+```java
+public class Sample {
+  public static void main(String[] args) throws IOException, InterruptedException {
+    TransactionFactory factory = TransactionFactory.create("<configuration file path>");
+    TwoPhaseCommitTransactionManager transactionManager =
+        factory.getTwoPhaseCommitTransactionManager();
+
+    int retryCount = 0;
+
+    while (true) {
+      if (retryCount > 0) {
+        // Retry the transaction three times maximum in this sample code
+        if (retryCount == 3) {
```

Comment:
[trivial] `retryCount >= 3` would be a little bit robust

Reply:
Fixed in 1041dda610d74ac00596e1ab1f69d6bbdeb32106.